infra/build/functions/build_project.py

```diff
@@ -510,6 +511,113 @@ def _indexer_built_image_name(name: str):
   return f'us-docker.pkg.dev/oss-fuzz/indexer/{name}'
 
 
+def _tracer_built_image_name(name: str):
+  # TODO(ochang): Write this to a tar (via docker image save) and upload this to
+  # GCS.
+  return f'us-docker.pkg.dev/oss-fuzz/tracer/{name}'
+
+
+def _create_indexed_build_steps(project,
+                                build,
+                                timestamp,
+                                env,
+                                build_type='indexer'):
+  """Creates the build steps for a specific indexer type."""
+  if build_type == 'indexer':
+    container_name = _INDEXED_CONTAINER_NAME
+    image_name = _indexer_built_image_name(project.name)
+    build_script_command = '/opt/indexer/index_build.py'
+    # Save the CDB fragments so we can re-use them for rebuilding indexes.
+    artifact_commands = ['cp -r $$OUT/cdb /cdb']
+  elif build_type == 'tracer':
+    container_name = _TRACING_CONTAINER_NAME
+    image_name = _tracer_built_image_name(project.name)
+    build_script_command = (
+        '/opt/indexer/index_build.py '
+        '--fuzzing-engine=tracing_engine.cc '
+        '--coverage-flags=-fsanitize-coverage=trace-pc-guard,func '
+        '--binaries-only')
+    artifact_commands = []
```

Comment:
any reason why this is different in the tracer case? 

We probably want to save CDB fragments in any case?

Reply:
It's not required, but if we want the tracer build to be the default for everything, we'll need it for re(indexing).